tests/Css/StyleTest.php

```diff
@@ -247,6 +248,8 @@ public static function cssImageWithStylesheetBaseHrefProvider(): array
      * @dataProvider cssImageNoBaseHrefProvider
      * @group regression
```

Comment:
Can you also add the `PHPUnit\Framework\Attributes\Group` attribute for the `@group` annotations?

Reply:
done 🎉